test/controllers/BankDetailsControllerSpec.scala

```diff
@@ -241,6 +241,8 @@ class BankDetailsControllerSpec extends ItSpec {
       AuthStub.authorise()
       EssttpBackend.ChosenTypeOfBankAccount.findJourney()
       EssttpBackend.DirectDebitDetails.updateDirectDebitDetails(TdAll.journeyId)
+      Bars.Validate.validateSuccess()
```

Comment:
Could we have a test for if Bars returns a not valid response, just checking that it doesn't redirect user to confirm direct debit details or something

Reply:
Yep. "Still todo: add tests" 🙂